### PR TITLE
make-admin-user-manage-question

### PR DESCRIPTION
**Pull request by Nghicv**

### CODE REVIEW

File: app/controllers/admin/words_controller.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+class Admin::WordsController < ApplicationController
+
+  before_action :logged_in_user
+  before_action :load_word, only: [:edit, :destroy, :update]
+
+  def index
+    @words = Word.paginate(page: params[:page],per_page: Settings.page_per).order t("name")
```
Comment (hellovietnam93) [inline app/controllers/admin/words_controller.rb]: sao lại dùng I18n chỗ này, ``` :name ```

File: app/controllers/admin/words_controller.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    if @word.save
+      flash[:success] = t "words.create_success"
+      redirect_to admin_words_path
+    else
+      render text: "error"
+    end
+  end
+
+  def destroy
+    @word.destroy
```
Comment (hellovietnam93) [inline app/controllers/admin/words_controller.rb]: fails message

File: app/controllers/admin/words_controller.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  def edit
+    @categories = Category.all
+  end
+
+  def update
+    if @word.update_attributes word_params
+      flash[:success] = t "flash.update"
+      redirect_to admin_words_path
+    else
+      render :edit
```
Comment (hellovietnam93) [inline app/controllers/admin/words_controller.rb]: fails message

File: app/views/admin/words/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            <th><%= t "tool" %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @words.each do |word| %>
+            <tr id="word-<%= word.id %>">
+              <td><%= word.name %></td>
+              <td><%= word.category.name %></td>
+              <% word.answers.each do |answer| %>
+                <% if answer.is_correct %>
```
Comment (hellovietnam93) [inline app/views/admin/words/index.html.erb]: refactor

File: app/views/admin/words/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+                  <td><%= answer.content %></td>
+                <% end %>
+              <% end %>
+              <% word.answers.each do |answer| %>
+                <% unless answer.is_correct %>
```
Comment (hellovietnam93) [inline app/views/admin/words/index.html.erb]: refactor

File: app/views/admin/words/_form.html.erb [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<%= form_for [:admin, @word] do |f| %>
+
+  <%= render "shared/error_messages", object: f.object %>
+
+  <%= f.label :name %>
+  <%= f.text_field :name, class: "form-control"%>
+
+  <div class="form-group">
+    <%= f.label :category_id %>
+    <%= f.select :category_id, options_for_select(@categories.map{ |c| [c.name, c.id ] },
```
Comment (dieunb) [inline app/views/admin/words/_form.html.erb]: convention

**Pull request merged**